lib/core/include/irods/connection_pool.hpp

```diff
@@ -15,6 +16,33 @@
 
 namespace irods
 {
+    /// Defines various options for changing the behavior of a connection pool.
+    ///
+    /// \since 4.3.1
+    struct connection_pool_options
+    {
+        /// Defines the number of API requests that can be sent via a connection before
+        /// refreshing it.
+        ///
+        /// Independent of any other option.
+        ///
+        /// \since 4.3.1
+        std::optional<std::int16_t> refresh_connections_after_n_retrievals;
+
+        /// Defines the number of seconds that must pass before refreshing a connection.
+        ///
+        /// Independent of any other option.
+        ///
+        /// \since 4.3.1
+        //std::optional<std::chrono::seconds> refresh_connections_after_n_seconds;
+
+        /// Instructs the connection pool to refresh a connection if changes to any resource
+        /// within the zone has been modified.
```

Comment:
Consider rephrasing this.

Reply:
instructs the connection pool to refresh a connection if there have been changes detected in any of the zone's resources.

Reply:
better...

'Instructs the connection pool to  refresh a connection when changes are detected in any of the zone's resources'

Reply:
or '...when a change is detected in any...'

Reply:
Done